### ISSUE TITLE
Define Accessibility Mappings for MathML Elements

### ISSUE THREAD

**Issue by Epigenetic** [w3c/mathml-core#196]
MathML does not currently have any mapping to ARIA. as a result, screen readers cannot process MathML markup and will not read anything out for that content. Currently chromium will output roles like MathMLNumber for `mn` while Firefox outputs "mathml number", neither of which are actual ARIA roles. Defining what these mappings should be is the first step to getting screen readers to be able to understand this content.

**Comment by fred-wang** [comment]
See also https://w3c.github.io/mathml-aam

**Comment by davidcarlisle** [comment]
see also the work on intent and accessible readings

https://w3c.github.io/mathml/#mixing_intent

https://github.com/w3c/mathml/issues?q=is%3Aissue+is%3Aopen+label%3Aintent

**Comment by bkardell** [comment]
The working group discussed this yesterday, minutes follow: 

> BK: This should be closed, here is why I held off just closing it. [explains some history]
> BK: Joanie, from Igalia and who was an aria co-chair, had spun up an AAM copy that was just for MathML. Has stepped down since and is mainly focusing on implementation things these days, Val Young, also from Igalia is a new co-chair and I talked to her about what to do with this. She also can't just take it on, and doesn't know MathML.  So, we needed a co-author. I spoke to Neil and he has agreed to be co-author on that spec. (mathml-core AAM).  Iwould like to close this issue. It does  not belong in MathML core, it has it's own repo... I've held off though because Neil hasn't been named co-author yet, nor Val (not sure if that one matters as much)... We were going to send a pull request for Neil to get added to the AAM?

> NS: I can add the request to add myself to the AAM. 

> **ACTION** ns submit for ns to bge an editor the the MathML AAM accessibility api mapping 
ns will close 196

**Comment by Steve-Noble** [comment]
I would be happy to help Neil with writing the spec, if that would be useful.
